index.js

```diff
@@ -1,22 +1,24 @@
 const express = require("express");
 const app = express();
-app.listen(3000, () => console.log("listening at 3000"));
+
+const port = process.env.PORT || 3000;
+
+app.listen(port, () =>
+    console.log(`Server running at http://localhost:${port}`)
+);
 app.use(express.static("public"));
 
 const Datastore = require("nedb-promises");
 const database = Datastore.create("database.db");
 
 app.get("/vote/:pollId/:choice", async (request, response) => {
-  const _id = request.params.pollId;
-  const choice = request.params.choice;
-  const poll = await database.findOne({ _id });
-  poll[choice]++;
-  await database.update({ _id }, poll);
-  response.send(poll);
+    const { pollId: _id, choice } = request.params;
+    await database.update({ _id }, { $inc: { [choice]: 1 } });
+    response.send(poll);
```

Comment:
This suggestion is just to make it work like before but could be commented as well since this line is not necessary (at least for now).

```suggestion
    response.send(await database.findOne({ _id }));
```

Reply:
The [update function](https://github.com/bajankristof/nedb-promises/blob/master/docs.md#Datastore+update) accepts an option `returnUpdatedDocs` so it can still be accomplished in one call. Included a new commit.

Reply:
That's even better... +1